app/controllers/tags_controller.rb

```diff
@@ -0,0 +1,38 @@
+class TagsController < ApplicationController
+  before_action :set_tag, only: [ :edit, :update, :destroy ]
+
+  def index
+    @tags = Current.user.tags.left_outer_joins(:taggings)
+      .group('tags.id')
+      .select('tags.*, COUNT(taggings.id) as taggings_count')
+      .order('taggings_count DESC, tags.name ASC')
```

Comment:
```suggestion
    @tags = Current.user.tags.
      .joins(:taggings)
      .group(:id)
      .select('tags.*, COUNT(taggings.id) as taggings_count')
      .order('taggings_count DESC', :name)
```

I think this would be cleaner. wdyt?